packages/wonder-stuff-testing/src/jest/isolate-modules.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+/**
+ * Isolate imports within a given action using jest.isolateModules.
+ *
+ * This is a helper for the `jest.isolateModules` API, allowing
+ * code to avoid the clunky closure syntax in their tests.
+ *
+ * @param {() => T} action The action that contains the isolated module imports.
+ * We do it this way so that any `require` calls are relative to the calling
+ * code and not this function. Note that we don't support promises here to
+ * discourage dynamic `import` use, which doesn't play well with standard
+ * jest yet.
+ */
+export const isolateModules = <T>(action: () => T): T => {
```

Comment:
This code is taken from the wonder-blocks-testing version, which this will ultimately replace.